.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,35 @@
+name: Bug Report
+description: Something not behaving the way it should? Let us know!
+title: "Bug: <Title>"
+labels: [bug, needs triage]
+assignees:
+  - Grossley
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: A clear description of what the bug is. Please consider adding pictures and/or videos, as they really help!
+    placeholder: |
+      When I do X then Y happens.
```

Comment:
A better description would be `X crashes` or something along those lines. The way it's right now, makes it very similar to reproducing instructions.